api/v1alpha2/groupversion_info.go

```diff
@@ -14,9 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha2 contains API Schema definitions for the image v1alpha2 API group
```

Comment:
Are these unnecessary?

Reply:
No, they are already declared in `v1alpha2/doc.go`.

Reply:
Got it. I seem to always end up in a situation where kubebuilder's markers are missing or in the wrong place, so I am over-vigilant about markers :-S